core/lib/dal/src/prover_dal.rs

```diff
@@ -314,6 +314,7 @@ impl ProverDal<'_, '_> {
                 r#"
                 SELECT COUNT(*) as "count!", circuit_type as "circuit_type!", status as "status!"
                 FROM prover_jobs
+                WHERE status <> 'skipped' and status <> 'successful' 
```

Comment:
I'm not sure it will improve things much. Instead, I suggest adding `WHERE l1_batch_number >= $last_proven_batch`

Reply:
I don't think it's better and it will take a bit more time to implement. 
we don't have this value at the start and now we can. 
Seems they are eventually close to each other. Let's keep this fix

Reply:
I'm also not sure if this will help, but Serghei thought it will. I'm also thinking `WHERE l1_batch_number >= $last_proven_batch` is better though - even if we need an additional query to determine it